README_new.md

```diff
@@ -0,0 +1,62 @@
+ - Languages (Swift, Objective-C) and platforms (iOS, watchOS, tvOS, macOS)
+ - Architectural design patterns (MVC, MVVM, Viper, etc)
+ - Knowledge of iOS APIs
+ - Knowledge of the development cycle
+ - Knowledge of reactive programming
+ - Interview demo project
+- Interest in sharing one's knowledge
+- Interest in learning new knowledge
+- Interest in keeping oneself up to date
+- Ability to think and work independently
+- Ability to work in a large team
+- Professional and personal maturity
+- Overall attitude
+- Likelihood of fitting in well with the company's culture
+- Is the candidate someone we would like to work with on a daily basis?
+
+### The iOS recruitment process
+
+Our recruitment process is different for candidates who are interested in working remotely and those who are interested in working on-site at our office in London, in **three** ways:
+- remote candidates are **required** to be proficient in **reactive programming** (`ReactiveSwift`/`ReactiveCocoa` or `RxSwift`) at an above-average level
+- remote candidates are **required** to use **reactive programming** in their interview demo projects
+- technical interview sessions differ in number and duration:
+    * **on-site**: initial call -> demo -> 3-hour face-to-face technical interview
+    * **remote**: initial call -> demo -> one or more Skype interviews -> 1 working day with us (at our expense)
+
+As indicated above, we expect all candidates to submit an ["interview demo project"](https://github.com/Babylonpartners/iOS-Interview-Demo/blob/open_sourced_recruitment/demo.md) but, in order to be accommodating of different schedules, we provide three different ways in which a candidate can submit a demo of their work. **Only one of these is required**.
+
+1. The babylon demo project
+2. A project you've already done
+3. An open source project you've done or contributed to
+
+A demo project should be an example of one's **best** work. We don't just want to see a technically good project; we want to be *wowed* by a candidate's techincal skills and creativity. If we then think the demo project is interesting enough, we'll proceed to a face-to-face step, either on Skype (for remote candidates) or in person (for on-site candidates).
+
+That face-to-face step typically starts with a discussion around the candidate's project. We normally ask the candidate to implement a new feature (even if it's just using pseudo-code), or to refactor and unit-test a particular component, and in general we aim to have an informal discussion about the candidate's approach.
+
+We may also focus on more technical aspects and ask a number of highly technical questions. As indicated earlier, we have open-sourced these questions as well since we believe that a fully open recruitment process gives all candidates a fairer chance to prepare themselves, and makes our assessment uniform across all candidates.
+
+Above all, be yourself and thank you for taking the time to apply to babylon health! 🌈
```

Comment:
nice

Reply:
Look at the commit message for the [commit that did it](https://github.com/Babylonpartners/iOS-Interview-Demo/pull/4/commits/07121b21d52edd41b15fed1e9860ecc2ac8d77cd).